Cargo.toml

```diff
@@ -1,17 +1,20 @@
 [package]
+authors = ["Richard Newman <rnewman@twinql.com>", "Nicholas Alexander <nalexander@mozilla.com>"]
 name = "mentat"
 version = "0.4.0"
-authors = ["Richard Newman <rnewman@twinql.com>", "Nicholas Alexander <nalexander@mozilla.com>"]
 
 [dependencies]
-rusqlite = "0.8.0"
 clap = "2.19.3"
+rusqlite = "0.8.0"
+slog = "1.4.0"
```

Comment:
I believe so.

http://softwareengineering.stackexchange.com/questions/321565/can-i-use-libraries-published-under-mpl-2-0-and-mit-in-a-project-that-uses-apl-2